tests/smoke-tests/test-balances-consistency.ts

```diff
@@ -297,19 +301,34 @@ describeSmokeSuite(`Verifying balances consistency...`, (context) => {
             .reduce((accumulator, curr) => accumulator + curr),
         },
       })),
-    ]
-      .flat()
-      .reduce((p, v) => {
-        if (!p[v.accountId]) {
-          p[v.accountId] = {
-            total: 0n,
-            reserved: {},
-          };
-        }
-        p[v.accountId].total += Object.keys(v.reserved).reduce((p, key) => p + v.reserved[key], 0n);
-        p[v.accountId].reserved = { ...p[v.accountId].reserved, ...v.reserved };
-        return p;
-      }, {} as { [key: string]: { total: bigint; reserved: { [key: string]: bigint } } });
+    ];
+
+    if (networkName === "Moonbase Stage") {
```

Comment:
This should not be conditional. If there are preimages it means we should include them. if it is empty (for other runtimes or older ones), it will work too